File: Assignment 9/program4.c
////////////////////////////////////////////////////////////////////////////////
//
//  Required Heder Files
//
////////////////////////////////////////////////////////////////////////////////
#include<stdio.h>        //For Input Output
////////////////////////////////////////////////////////////////////////////////
//
//  Function Name : RangeSum
//  Description   : It is used to return addition of all even number between given range  
//  Input         : Intiger  
//  OutPut        : Intiger  
//  Author        : Pratibha Sagar Pawar
//  Date          : 29 Oct 2025
//      
////////////////////////////////////////////////////////////////////////////////
int RangeSum(int iStart, int iEnd)
{
    
    if(iStart > iEnd)
    {
        printf("Invalid range");
        return 0;
    }

    int iCnt = 0;
    int iSum = 0;
    for(iCnt = iStart; iCnt<= iEnd ; iCnt++)
    {
        if(iCnt%2==0)
        {
            iSum = iSum +iCnt;
        }
    }
    return iSum;
}
////////////////////////////////////////////////////////////////////////////////
//
//  Entery point function for the application 
// 
////////////////////////////////////////////////////////////////////////////////
int main()
{
    int iValue1 =0,  iValue2 = 0;
    int iRet = 0;

    printf("Enter Starting point ");
    scanf("%d",&iValue1);

    printf("Enter Ending point ");
    scanf("%d",&iValue2);

    iRet=RangeSum(iValue1,iValue2);

    printf("Addition is %d",iRet);

    return 0;

}
//  End of main Function
////////////////////////////////////////////////////////////////////////////////
//
//  Input : 23 30   Output : 108
//  Input : 10 18   Output : 70
//  Inpur : -10 2   Output : Invalid Range
//
////////////////////////////////////////////////////////////////////////////////

File: Assignment 32/program4.java
////////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

////////////////////////////////////////////////////////////////////////////////
//
//  Class Name    : class pattern
//  Description   : It is used to Display pattern 
//  Input         : Intiger  
//  OutPut        : Pattern 
//  Author        : Pratibha Sagar Pawar
//  Date          : 22 Nov 2025
//      
////////////////////////////////////////////////////////////////////////////////
class pattern
{
    public void Display(int iRow, int iCol)
    {
        int i = 0;
        int j = 0;
       
            for(i = 1; i<=iRow; i++ )
        {
            for(j = 1; j<=iCol; j++)
            {
                if( i + j == iRow +1 || i == iRow || i==1 ||j==1 || j == iCol)
                {
                    System.out.print("*\t");                }
                else if(i+j>=iRow+2)
                {
                   System.out.print("$\t");
                }
                else
                {
                   System.out.print("#\t");

                }
                
        
            }   System.out.println();
        }
        System.out.println();
    }
}
///////////////////////////////////////////////////////////////////////////////
//
//  Entery point method for the application 
// 
////////////////////////////////////////////////////////////////////////////////

class program4
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue1 = 0;
        int iValue2 = 0;

        System.out.println("Enter the number of Rows : ");
        iValue1 = sobj.nextInt();

        System.out.println("Enter the number of columns : ");
        iValue2 = sobj.nextInt();

        pattern pobj = new pattern();
        pobj.Display(iValue1, iValue2);

    }
}
//  End of main method
////////////////////////////////////////////////////////////////////////////////
//
//  Input  : iRow = 5  iCol = 5
//
//  Output :  *    *    *    *    *
//            *    #    #    *    *
//            *    #    *    $    *
//            *    *    $    $    *
//            *    *    *    *    *
////////////////////////////////////////////////////////////////////////////////


